cosmoshub/chain.json

```diff
@@ -288,6 +288,10 @@
       {
         "address": "cosmos-grpc.polkachu.com:14990",
         "provider": "Polkachu"
+      },
+      {
+        "address": "https://cosmos-api.cyphercore.io",
```

Comment:
This is the grpc definition, rest API instead?

Reply:
@thomasWos Fixed. Thanks for catching that.